app/account/views.py

```diff
@@ -250,3 +250,8 @@ def unconfirmed():
     if current_user.is_anonymous() or current_user.confirmed:
         return redirect(url_for('main.index'))
     return render_template('account/unconfirmed.html')
+
+
+@account.route('/profile')
```

Comment:
We are going to want an `@login_required` decorator on this page. Checkout some of the other routes in the file for examples.